multichain-aggregator/multichain-aggregator-migration/src/m20241031_064924_add_api_keys.rs

```diff
@@ -0,0 +1,28 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        let sql = r#"
+        CREATE TABLE api_keys (
+            id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+            key UUID NOT NULL UNIQUE,
+            chain_id bigint NOT NULL REFERENCES chains (id),
+            created_at TIMESTAMP NOT NULL DEFAULT (CURRENT_TIMESTAMP)
```

Comment:
let's merge everything in one migration since this is initial PR and there is no database with applied migrations

Reply:
Good idea, thank you